manifests/base/manager/manager.yaml

```diff
@@ -64,7 +74,7 @@ spec:
   ports:
     - name: monitoring-port
       port: 8080
-      targetPort: 8080
+      targetPort: 8443
```

Comment:
Do we require to add annotations for prometheus like we did in v1?
https://github.com/kubeflow/trainer/blob/release-1.9/manifests/base/service.yaml#L5C5-L7